docs/UserGuide.adoc

```diff
@@ -246,14 +251,15 @@ Increases the debt of the 2nd person by $234.
 // tag::payback[]
 === Decreasing the debt of a person: `payback`
 
-Decrease the debt of a person by the amount entered. +
+Decreases the debt of a person by the amount entered. +
 Format: `payback [INDEX] AMOUNT`
 
 ****
 * Decreases the debt of the person at the specified `INDEX` by `AMOUNT`. The index refers to the index number shown in the last person listing. The index *must be a positive integer* 1, 2, 3, ...
-* If no index is specified, the debt of the currently selected person is decreased instead.
-* `AMOUNT` has to be in dollars and cents. For example: `600` which represents $600.
-* `AMOUNT` repaid cannot be more than the debt owed by the person at the specifiec `INDEX`
+* If no index is specified, the debt of the currently selected person is updated instead.
+* `AMOUNT` has to be in dollars and cents. For example: `600.15` which represents $600.15.
+* `AMOUNT` repaid cannot be more than the debt owed by the person at the specifiec `INDEX`.
+* If a person fully repays his/her debts, the date repaid will be set to the day that this command was entered. Also, the person will be listed in the whitelist.
```

Comment:
not always. if a blacklisted person does this he/she will not be in the whitelist.
consider changing to "in the whitelist if he/she is not banned" or "not blacklisted"

Reply:
corrected